src/components/Addbook.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './navbar.css';
+
+function AddBook() {
+  return (
+    <>
+      <div className="bookcont">
+        <div className="books-info">
+          <span className="book-cata">Action</span>
+          <h3 className="book-title">The Hunger Games</h3>
+          <p className="book-author">Suzanne Collins</p>
+          <button type="button" className="btn" id="commit">
+            Comments
+          </button>
+          <button type="button" className="btn" id="remove">
+            Remove
+          </button>
+          <button type="button" className="btn" id="edit">
+            Edit
+          </button>
+        </div>
+        <div className="compelet">
+          <span className="precent">64%</span>
+          <h5>completed</h5>
+        </div>
+        <div className="current-part">
+          <p className="current-chapter">CURRENT CHAPTER</p>
+          <h4>Chapter 17</h4>
+          <button type="button" className="upd-btn">
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
+
+      {/* add book part */}
+
+      <div className="contanier">
+        <h2>ADD NEW BOOK</h2>
+        <div className="cont">
+          <input type="text" placeholder="Book Tilte" className="input" />
+          <select className="select-cata">
+            <option selected>Category</option>
+            <option>Fiction</option>
+            <option>Genral</option>
+            <option>Amazing</option>
+          </select>
+          <button type="button" className="addbtn">
+            ADD BOOK
+          </button>
+        </div>
+      </div>
+    </>
+  );
+}
+
+export default AddBook;
```

Comment:
Great progress on your project so far! 👍🏿 However, in line with the project requirements, you are tasked with developing a book component, which includes a booklist component. This booklist component should receive book data as a prop and display the title and author of each book. Additionally, it should feature a button for removing books.

Please note that the button's functionality for removing books is currently a placeholder and is not yet operational.

<NewBookForm>
This component is responsible for rendering a form containing input fields for both the book's title and author. It also includes a submit button.

Similar to the previous button, the submit button's functionality for adding books is also a placeholder and is not yet functional.